build-logic/jvm/src/main/kotlin/gradlebuild.unittest-and-compile.gradle.kts

```diff
@@ -230,7 +230,12 @@ fun Test.configureRerun() {
     }
 }
 
-fun Test.determineMaxRetry() = if (project.name in listOf("smoke-test", "performance", "build-scan-performance")) 1 else 2
+fun Project.isFlakyTestQuarantine() = providers.gradleProperty("flakyTestQuarantine").isPresent
```

Comment:
This property should go into `BuildParams`.

Reply:
Done.